test/acceptance/view_benchmark_graphs_test.rb

```diff
@@ -72,9 +72,60 @@ class ViewBenchmarkGraphsTest < AcceptanceTest
     assert_not page.has_content?("#{benchmark_run_category_humanize} memory Graph")
   end
 
+  test "User should be able to view long running benchmark graphs when they go back and forward 
+    in the broswer window" do
+
+    javascript_driver = Capybara.javascript_driver
+    default_driver = Capybara.current_driver
+    Capybara.javascript_driver = :selenium
+    Capybara.current_driver = :selenium
```

Comment:
i tried it without selenium test, and it works sometimes, but not the other times. It doesn't seem to wait for the full rendering of the graph before it starts checking for the DOM.